### PR TITLE
Fixes #13199 - change pulp_parent to qpid_client

### PR DESCRIPTION
**Pull request by johnpmitsch**

### CODE REVIEW

**Comment by stbenjam** [comment]
[test]

**Comment by johnpmitsch** [comment]
This requires https://github.com/Katello/puppet-certs/pull/78

**Comment by johnpmitsch** [comment]
I ran foreman-installer --scenario katello on an existing box with this and the certs PR changes (mentioned above)  and the installer ran fine except for a seed error that I am pretty sure we already know about and is unrelated. I was able to create and sync a repo as well.

File: manifests/init.pp [hunk 1]
```diff
@@ -103,14 +103,14 @@
...
-  class { '::certs::pulp_parent': } ~>
+  class { '::certs::qpid_client': } ~>
   class { '::pulp':
     oauth_enabled          => true,
     oauth_key              => $katello::oauth_key,
     oauth_secret           => $katello::oauth_secret,
     messaging_url          => "ssl://${::fqdn}:5671",
-    messaging_ca_cert      => $certs::pulp_parent::messaging_ca_cert,
-    messaging_client_cert  => $certs::pulp_parent::messaging_client_cert,
+    messaging_ca_cert      => $certs::qpid_client::messaging_ca_cert,
```
Comment (ehelms) [inline manifests/init.pp]: This should just be `$::certs::ca_cert` now.

**Comment by johnpmitsch** [comment]
@ehelms updated

**Comment by johnpmitsch** [comment]
@ehelms I'm not sure how to fix these broken tests, are there fixture files?

**Comment by stbenjam** [comment]
I think once https://github.com/Katello/puppet-certs/pull/78 gets merged your tests should pass

**Comment by ehelms** [comment]
Can you please update the minimum here to 0.5.0 -- https://github.com/Katello/puppet-katello/blob/master/metadata.json#L37

**Comment by johnpmitsch** [comment]
@ehelms updated

**Comment by ehelms** [comment]
Thanks @johnpmitsch !

**Pull request merged**